Source/WebCore/Modules/webauthn/AuthenticatorCoordinator.cpp

```diff
@@ -177,6 +177,22 @@ void AuthenticatorCoordinator::create(const Document& document, CredentialCreati
         return;
     }
 
+    if (createOptions.signal) {
+        createOptions.signal->addAlgorithm([weakThis = WeakPtr { *this }](JSC::JSValue) mutable {
+            m_isCancelling = true;
```

Comment:
I don't think this is going to build. Put this below the check and access it with weakThis->m_isCancelling = true.